tools/get.js

```diff
@@ -0,0 +1,98 @@
+var crypto = require('crypto');
+var moment = require('moment');
+
+function toLong(ip) {
+    var ipl = 0;
+    ip.split('.').forEach(function(octet) {
+        ipl <<= 8;
+        ipl += parseInt(octet);
+    });
+    return ipl >>> 0;
+}
+
+module.exports = {
+    toLong,
+    md5(x) {
+        return crypto
+            .createHash('md5')
+            .update(x)
+            .digest('hex');
+    },
+    nameToTime(x) {
+        (x = x.split('.')[0].split('T')), (x[1] = x[1].replace(/-/g, ':'));
+        x = x.join(' ');
+        return x;
+    },
+    ratio(width, height, ratio) {
```

Comment:
Can we maybe do some maths? Or is that over thinking it? Just thinking of other ratios like 16:10, or whatever may come of the future.
```
var gcd = function(a, b) {
    if ( ! b) {
        return a;
    }

    return gcd(b, a % b);
};

var ratio = function(height, width){
	var divisor = gcd(height, width);
  return height/divisor + ':' + width/divisor;
}
```